src/p_pspr.c

```diff
@@ -50,14 +50,14 @@ static struct
   int pitch;
 } recoil_values[] = {    // phares
```

Comment:
Similarly, if you want the recoil and "centering" values (1 and 8, plus this array) to be faithful to the original values, you'll have to do the proper angle to shearing height conversion instead of fudging the numbers.

Reply:
Would appreciate proper math, but I don't think it's worth much effort.

Reply:
I don't mind looking at it in a different PR.